### ISSUE TITLE
Loss function during training

### ISSUE THREAD

**Issue by yld1028** [Tombs98/MHNet#6]
Hello,

I hope you’re doing well. I have a question regarding the training process described in your article. I followed the instructions and started training the model, but I noticed that the loss function initially becomes negative instead of positive. I’m not sure if this is expected or if there might be an error in my implementation.

I have carefully reviewed the documentation and double-checked my code, but I haven’t been able to find a solution. Could you please help me understand if a negative loss value at the beginning of the training process is correct or not?

Thank you so much for your time and expertise.

**Comment by Tombs98** [comment]
> Hello,
> 
> I hope you’re doing well. I have a question regarding the training process described in your article. I followed the instructions and started training the model, but I noticed that the loss function initially becomes negative instead of positive. I’m not sure if this is expected or if there might be an error in my implementation.
> 
> I have carefully reviewed the documentation and double-checked my code, but I haven’t been able to find a solution. Could you please help me understand if a negative loss value at the beginning of the training process is correct or not?
> 
> Thank you so much for your time and expertise.

In this work, we use PSNR loss following NAFNet. It will get smaller and smaller. You can see https://github.com/megvii-research/NAFNet/issues/48#issuecomment-1195042862 for more informations
